recipes/bandit/all/conanfile.py

```diff
@@ -0,0 +1,40 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.51.3"
+
+
+class BanditConan(ConanFile):
+    name = "bandit"
+    description = "Human-friendly unit testing for C++11"
+    topics = ("testing", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/banditcpp/bandit"
+    license = "MIT"
+    no_copy_source = True
```

Comment:
```suggestion
```
As we need to apply a patch, let's avoid doing it over the source folder. For debugging and development, it's a mess for us.